tests/testthat/test-linl.R

```diff
@@ -0,0 +1,32 @@
+context("Letter Options")
+
+test_that("Base letter", {
+    frontmatter$plainquote <- TRUE
```

Comment:
I meant for all of these tests to be skipped on CRAN but missed this one:
```suggestion
    skip_on_cran()
    frontmatter$plainquote <- TRUE
```